### PR TITLE
Update LIP 0062

### PR DESCRIPTION
**Pull request by MaximeGagnebin**
Update LIP 62 to encompass all functions defined in LIP 38.

### CODE REVIEW

**Review by Incede — APPROVED** [review]
Logic looks good, optional syntactical suggestion though.

**Review by sergeyshemyakov — APPROVED** [review]
Is good!

File: proposals/lip-0062.md [hunk 1]
```diff
@@ -67,6 +67,8 @@ For the BLS signature scheme [used in Lisk](https://github.com/LiskHQ/lips/blob/
 
 The `signBLS` and `verifyBLS` functions defined in [LIP 0038](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0038.md#signing-and-verifying) are superseded by the functions defined in this LIP.
 
+The functions `verifyAggSig` and `verifyWeightedAggSig` defined in [LIP 0038](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0038.md#aggregate-signatures-and-their-verification) must also be updated to hash the tagged message before calling the `FastAggregateVerify` function.
```
[the remark this answers is not in the document]
Comment (MaximeGagnebin) [reply proposals/lip-0062.md]: I applied the small style suggestion.
For the wording `supersedes`, I see what you mean, a small difference is that those 2 functions have a logic that I don't want to fully redefine here. So I would prefer to keep the current wording, if that's ok.
I think that linking twice is fine, especially that we link the specific sections.

File: proposals/lip-0062.md [hunk 2]
```diff
@@ -67,6 +67,8 @@ For the BLS signature scheme [used in Lisk](https://github.com/LiskHQ/lips/blob/
 
 The `signBLS` and `verifyBLS` functions defined in [LIP 0038](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0038.md#signing-and-verifying) are superseded by the functions defined in this LIP.
 
+The `verifyAggSig` and `verifyWeightedAggSig` functions defined in [LIP 0038](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0038.md#aggregate-signatures-and-their-verification) must also be updated to hash the tagged message before calling the `FastAggregateVerify` function.
+
```
Comment (gkoumout) [inline proposals/lip-0062.md:71]:
```suggestion
If I understand well LIPs 037 and 038, the modification in all 4 functions is *exactly* the same. Just add a line   `hashedMessage = hash(taggedMessage)` and in their return replace taggedMessage by hashedMessage. So they should be treated same way in text, IMO. 
```

**Review by gkoumout — APPROVED** [review]
Looks good to me! Added a comment, feel free to ignore it.

**Review by mitsuaki-u — APPROVED** [review]

**Review by karmacoma — APPROVED** [review]

**Pull request merged**